README.md

```diff
@@ -91,10 +91,10 @@ available Cloud Run regions](https://cloud.google.com/run/docs/locations))
 
 The rollout strategy consists of the steps and health criteria.
 
-- `-cli-run-interval`: The time between each health check, in seconds (default:
-`60`). This is only need it if running with `-cli` option.
-- `-healthcheck-offset`: To evaluate the candidate's health, use metrics from
-the last `N` minutes relative to current rollout process (default: `30`)
+- `-cli-run-interval`: The time between each health check (default: `60s`). This
+is only need it if running with `-cli` option.
+- `-healthcheck-offset`:range of time for retrieving metrics relative to current
+rollout process (default: `30m`)
 - `-min-requests`: The minimum number of requests needed to determine the
```

Comment:
probably make a note that 

> [...] needed (with -healthcheck-offset) to determine

also worth adding to cli help

Reply:
What exactly do you mean by this? What's needed with `healthcheck-offset`?

Reply:
I meant that it is worth noting that min-requests are calculated "within $healthcheck-offset minutes" and not all-time.

Reply:
Oh sure. Let me add that piece